### PR TITLE
Add training to CLI (and various other related issues)

### LINKED ISSUE #11: rework cli for training and classifying
Small to-do for `classify` CLI:
- [ ] change `model` to `model_path`

Write `train` CLI:
- [ ] options for seed, number of reads to simulate, train_ratio, batch_size, gpu, summary_interval, epochs, summarize, learning_rate, log_dir, concise=True, config file
- [ ] essentially implement what is in the trial scripts
- [ ] move trial_ids and validation ids to files
- [ ] find way to test

### LINKED ISSUE #12: consider migrating data downloading and label preparation to `taxa-assign-benchmarking`

### LINKED ISSUE #15: Remove automatic class labeling
Let it be passed into the trainer functionality

### LINKED ISSUE #16: Add some basic test functionality for `trainer`
This is required so that I can be confident in changes made for #15

### LINKED ISSUE #18: deprecate scripts that involve `id_to_lineage`
This will help remove the dependence upon ete3

### PR DESCRIPTION
**Pull request by gwarmstrong**
This is progress on #11 

Part of this rework needs updates to the `trainer` function #15 

Fixes #11 Fixes #15 Fixes #16 Fixes #18 Closes #12

### CODE REVIEW

**Comment by gwarmstrong** [comment]
^ the above should handle #16
